quixstreams/app.py

```diff
@@ -431,6 +441,90 @@ def stop(self):
         """
         self._running = False
 
+    def get_producer(self) -> Producer:
+        """
+        Create and return a pre-configured Producer instance.
+        The Producer is initialized with params passed to Application.
+
+        It's useful for producing data to Kafka outside the standard Application processing flow,
+        (e.g. to produce test data into a topic).
+        Using this within the StreamingDataFrame functions is not recommended, as it creates a new Producer
+        instance each time, which is not optimized for repeated use in a streaming pipeline.
+
+        Example Snippet:
+
+        ```python
+        from quixstreams import Application
+
+        app = Application.Quix(...)
+        topic = app.topic("input")
+
+        with app.get_producer() as producer:
+            for i in range(100):
+                producer.produce(topic=topic.name, key=b"key", value=b"value")
```

Comment:
small QoL thing, but I feel like the user's should just get to use `Topic` objects instead of having to call `Topic.name`. Maybe we want to have a small subclass of producer with this behavior? Kind of overlaps with the `RowProducer` though, which I know is also due for an overhaul, and maybe we just use that instead?